Console/Command/NostoAccountConnectCommand.php

```diff
@@ -213,7 +213,7 @@ private function updateNostoTokens(array $tokens, $accountId, SymfonyStyle $io,
      */
     private function getTokensFromInput(InputInterface $input, SymfonyStyle $io)
     {
```

Comment:
Is there a way to get rid of these warning? 🤔 @supercid

Reply:
Yeah, we can configure the sniffer later on, this PR is already too long